src/fs/host-default.test.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @vitest-environment jsdom
+ * @vitest-environment-options { "url": "http://localhost:3000" }
+ */
+import { fromByteArray } from "base64-js";
+import { MAIN_FILE } from "./fs";
+import { DefaultHost } from "./host";
+import { defaultInitialProject } from "./initial-project";
+import { testMigrationUrl } from "./migration-test-data";
+
+describe("DefaultHost", () => {
+  it.skip("uses migration if available", async () => {
```

Comment:
Is this skipping on purpose or can it be reinstated?

Reply:
Nope, same issue as Grace's comment but different test. Fixed.